src/lib/all_parameter_variant.hpp

```diff
@@ -18,8 +19,9 @@ namespace hana = boost::hana;
 // This holds pairs of all types and their respective string representation
 static constexpr auto parameter_types =
     hana::make_tuple(hana::make_pair("AllTypeVariant", hana::type_c<AllTypeVariant>),
-                     hana::make_pair("ColumnID", hana::type_c<ColumnID>),              // NOLINT
-                     hana::make_pair("Placeholder", hana::type_c<ValuePlaceholder>));  // NOLINT
+                     hana::make_pair("ColumnID", hana::type_c<ColumnID>),                // NOLINT
+                     hana::make_pair("LQPColumnOrigin", hana::type_c<LQPColumnOrigin>),  // NOLINT
```

Comment:
Don't like LQPColumnOrigin as a name either, LogicalColumnReference sounds nice. Would refrain from PhysicalColumnID because the type is used in the LQP as well

Reply:
BUT: LogicalColumnReference doesn't make that much sense. What is a "Logical" Column Reference? LQPColumnReference comes to mind, what do you think about that? @mrks

Reply:
(Keep in mind that LQPColumnOrigins are not supposed to have a node member pointing to an arbitrary node, but to the specific node that created the column.)

Reply:
works for me

Reply:
Proposal: Let's call it `LQPColumnReference` with members `origin_node` and `origin_column_id` - that makes semantics the most clear, imo.

Reply:
ok